packages/framework-core/src/booster-graphql-dispatcher.ts

```diff
@@ -28,36 +29,38 @@ export class BoosterGraphQLDispatcher {
 
     this.graphQLSchema = new GraphQLGenerator(config, commandDispatcher, this.readModelDispatcher).generateSchema()
     this.websocketHandler = new GraphQLWebsocketHandler(
+      config,
       logger,
-      this.config.provider.readModels.notifySubscription.bind(null, config),
+      this.config.provider.auth,
+      this.config.provider.connections,
       {
         onStartOperation: this.runGraphQLOperation.bind(this),
         onStopOperation: this.readModelDispatcher.unsubscribe.bind(this.readModelDispatcher),
-        onTerminateOperations: this.readModelDispatcher.unsubscribeAll.bind(this.readModelDispatcher),
+        onTerminate: this.handleDisconnect.bind(this),
       }
     )
   }
 
   // eslint-disable-next-line @typescript-eslint/no-explicit-any
   public async dispatch(request: any): Promise<any> {
```

Comment:
Maybe this is out of scope for this PR, but could you replace these `any` with `unknown`?

Reply:
Yeah! this makes total sense because we do nothing with this type. Thanks, I'll see if I can change more any's by unknown

Reply:
I took the liberty and replaced a lot of any's by unknown in the core package 😎
Check 1c063b7602e4a54e5510d82ed5d531c2eed8fe32

Reply:
And this 418308b1b7e182cbc60aa08ee79fdb7b36ffae39